### ISSUE TITLE
Exception importing large project

### ISSUE THREAD

**Issue by cursiveclojure** [cursiveclojure/cursive#399]
Reported by @sfnelson via email:

Importing our giant project with many modules (including checkouts)

Import seemed to work fine, but the following exception was thrown:

```
1:11:14 PM Throwable
           Too many events posted
           java.lang.Throwable
           at com.intellij.openapi.diagnostic.FrequentEventDetector.eventHappened(FrequentEventDetector.java:61)
           at com.intellij.openapi.vfs.newvfs.RefreshQueueImpl.a(RefreshQueueImpl.java:96)
           at com.intellij.openapi.vfs.newvfs.RefreshQueueImpl.execute(RefreshQueueImpl.java:65)
           at com.intellij.openapi.vfs.newvfs.RefreshSessionImpl.launch(RefreshSessionImpl.java:107)
           at com.intellij.openapi.vfs.newvfs.RefreshQueue.refresh(RefreshQueue.java:56)
           at com.intellij.openapi.vfs.newvfs.RefreshQueue.refresh(RefreshQueue.java:42)
           at com.intellij.openapi.vfs.newvfs.NewVirtualFile.refresh(NewVirtualFile.java:82)
           at com.intellij.openapi.vfs.VirtualFile.refresh(VirtualFile.java:670)
           at cursive.leiningen.project.wizard.LeiningenProjectImportStep$2.visitFile(LeiningenProjectImportStep.java:111)
           at com.intellij.openapi.vfs.VirtualFileVisitor.visitFileEx(VirtualFileVisitor.java:124)
           at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:243)
           at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:275)
           at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:275)
           at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:275)
           at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:295)
           at cursive.leiningen.project.wizard.LeiningenProjectImportStep.findProjectFiles(LeiningenProjectImportStep.java:98)
           at cursive.leiningen.project.wizard.LeiningenProjectImportStep.access$200(LeiningenProjectImportStep.java:35)
           at cursive.leiningen.project.wizard.LeiningenProjectImportStep$1.run(LeiningenProjectImportStep.java:73)
           at com.intellij.openapi.progress.impl.ProgressManagerImpl$TaskRunnable.run(ProgressManagerImpl.java:471)
           at com.intellij.openapi.progress.impl.ProgressManagerImpl$6.run(ProgressManagerImpl.java:281)
           at com.intellij.openapi.progress.impl.ProgressManagerImpl$2.run(ProgressManagerImpl.java:178)
           at com.intellij.openapi.progress.ProgressManager.executeProcessUnderProgress(ProgressManager.java:209)
           at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:212)
           at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcess(ProgressManagerImpl.java:171)
           at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:645)
           at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:419)
           at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
           at java.util.concurrent.FutureTask.run(FutureTask.java:266)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
           at java.lang.Thread.run(Thread.java:744)
           at com.intellij.openapi.application.impl.ApplicationImpl$1$1.run(ApplicationImpl.java:149)
```

**Comment by cursiveclojure** [comment]
Is this still an issue?

**Comment by sfnelson** [comment]
Yes. Do you want a fresh stack trace?

**Comment by cursiveclojure** [comment]
Not unless it's changed - just checking that the issue was still present.

**Comment by cursiveclojure** [comment]
This was fixed in 0.1.63
